design-library/source/_patterns/01-atoms/09-social-icon/_social-icon.scss

```diff
@@ -1,17 +1,27 @@
-li.social_media--item {
+.social-icon {
   display: inline-block;
   padding: 0;
-  margin: 0;
+  margin: 0 .5em;
   height: 2em;
   width: 2em;
 
-}
-
-a.social_media--icon {
-  display: block;
-  text-indent: -9999px;
-  height: inherit;
-  width: inherit;
-  background-repeat: no-repeat;
-  background-size: contain;
+  a {
+    background-repeat: no-repeat;
+    background-size: contain;
+    display: block;
+    height: inherit;
+    text-indent: -9999px;
+    display: block;
+    text-indent: -9999px;
+    width: inherit;
+    &.Twitter {
+      background-image: url('../images/social_media_icons/social_twitter.png');
```

Comment:
we may need to be careful of paths to images/files as this could complicate the Drupal processing. something to keep an eye on as we move forward :)